### PR TITLE
New package: AlexandreRouma.SDRPlusPlus version 1.0.4

### LINKED ISSUE #111415: [Package Request]: SDRPlusPlus
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/AlexandreRouma/SDRPlusPlus/releases
Publisher: AlexandreRouma
Package Name: SDRPlusPlus
Description: The bloat-free SDR software.
Package Version: 1.0.4
Installer URL: https://github.com/AlexandreRouma/SDRPlusPlus/releases/download/1.0.4/sdrpp_windows_x64.zip

### PR DESCRIPTION
**Pull request by CoolPlayLin**
Resolve #111415

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111880&drop=dogfoodAlpha

### CODE REVIEW

**Comment by CoolPlayLin** [comment]
?

**Comment by stephengillie** [comment]
1. This package has a dependency on `Microsoft.VCRedist.2015+.x64`
2. With 1. added, it errors with: 
> Faulting application name: sdrpp.exe, version: 0.0.0.0, time stamp: 0x616ca8cd
> Faulting module name: ucrtbase.dll, version: 10.0.19041.789, time stamp: 0x2bd748bf
Exception code: 0xc0000409
Fault offset: 0x000000000007286e
> Faulting process id: 0x240
> Faulting application start time: 0x01d9b4f20d654271
> Faulting application path: C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\AlexandreRouma.SDRPlusPlus__DefaultSource\sdrpp_windows_x64\sdrpp.exe
> Faulting module path: C:\Windows\System32\ucrtbase.dll
Report Id: 0eabe131-8343-420c-aa98-cf01b9c15959
> Faulting package full name: 
> Faulting package-relative application ID:

File: manifests/a/AlexandreRouma/SDRPlusPlus/1.0.4/AlexandreRouma.SDRPlusPlus.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: AlexandreRouma.SDRPlusPlus
+PackageVersion: 1.0.4
+Installers:
```
Comment (stephengillie) [inline manifests/a/AlexandreRouma/SDRPlusPlus/1.0.4/AlexandreRouma.SDRPlusPlus.installer.yaml:6]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Comment by CoolPlayLin** [comment]
I have no idea to fix

**Comment by CoolPlayLin** [comment]
What should I do to fix it?

**Comment by stephengillie** [comment]
Hi @CoolPlayLin,

This package installs normally, but doesn't appear to perform any function when launched. Is this expected?

image

**Comment by CoolPlayLin** [comment]
> Hi @CoolPlayLin,
> 
> This package installs normally, but doesn't appear to perform any function when launched. Is this expected?
> 
> image
> 

I don't know
CC @AlexandreRouma

**Comment by AlexandreRouma** [comment]
I don't support winget, I don't know who created the package but I'm not gonna spend time debugging this.

**Comment by CoolPlayLin** [comment]
please close this PR and the issue
CC @stephengillie

**Comment by AsciiWolf** [comment]
> This package installs normally, but doesn't appear to perform any function when launched. Is this expected?
Yes, this is a correct behavior if you run SDRPlusPlus inside a VM without 3D acceleration available. The app will most likely work fine when run on a physical host.

**Comment by AsciiWolf** [comment]
> please close this PR and the issue

Why? There is no need to close this PR if the original app developer does not support WinGet. It still can be merged if it works without issue (see my previous comment).

**Comment by CoolPlayLin** [comment]
> 
> Yes, this is a correct behavior if you run SDRPlusPlus inside a VM without 3D acceleration available. The app will most likely work fine when run on a physical host.

So, this software required specific hardware?

**Comment by AsciiWolf** [comment]
> So, this software required specific hardware?

No, it requires regular hardware (with full 3D acceleration) to run. Not a limited VM.

But yeah, since it is a SDR application, it also requires an actual Software Defined Radio hardware (for example the cheap [RTL-SDR dongle](https://www.rtl-sdr.com/buy-rtl-sdr-dvb-t-dongles/)) or many other to function.

> If so, this PR will be blocked with Hardware label

Why? There are already many software packages available on WinGet that are utilizing specific hardware parts.

**Comment by CoolPlayLin** [comment]
So, this software could run in any machine with win10 or win11?

**Comment by AsciiWolf** [comment]
Yes, it should work on any Win 10 / Win 11 machine as long as there is a 3D acceleration (and OpenGL) working.

**Comment by stephengillie** [comment]
> > Yes, this is a correct behavior if you run SDRPlusPlus inside a VM without 3D acceleration available. The app will most likely work fine when run on a physical host.
> 
> [AlexandreRouma/SDRPlusPlus#1151](https://github.com/AlexandreRouma/SDRPlusPlus/issues/1151)

Wow, this is an excellent reply. Thanks for confirming expected behavior.

**Comment by AsciiWolf** [comment]
The pipeline failure seems unrelated. Is it possible to re-run it? Thanks!

**Comment by AsciiWolf** [comment]
Is everything ok now? It looks like all checks have passed.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**